src/Compilers/CSharp/Portable/Symbols/Source/SourceFixedFieldSymbol.cs

```diff
@@ -55,8 +55,7 @@ public sealed override int FixedSize
                     DiagnosticBag diagnostics = DiagnosticBag.GetInstance();
                     int size = 0;
 
-                    VariableDeclaratorSyntax declarator = this.VariableDeclaratorNode;
-
+                    VariableDeclaratorSyntax declarator = VariableDeclaratorNode;
```

Comment:
>VariableDeclaratorSyntax declarator = VariableDeclaratorNode; [](start = 20, length = 61)

It looks like there is no meaningful change in this file. Consider reverting it to the previous state to reduce unnecessary churn in the code base.